task4-1/binaryAddition.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#define INT_SIZE (sizeof(int) * 8)
+// print a number in binary notation to the console
+void printBinaryNumber(int *num);
+// converting a number from decimal to binary
+void decimalToBinary(int num, int *convertedNum);
```

Comment:
Надо пустые строки между всеми объявлениями:
```suggestion
#define INT_SIZE (sizeof(int) * 8)

// print a number in binary notation to the console
void printBinaryNumber(int *num);

// converting a number from decimal to binary
void decimalToBinary(int num, int *convertedNum);
```